web/src/containers/QuickExchange/index.tsx

```diff
@@ -85,7 +88,7 @@ export const QuickExchangeContainer = () => {
         const seconds = +msPricesUpdates() / 1000;
         setTime(ssToMMSS(seconds));
 
-        if (abilities.read && !CanCan.checkAbilityByAction('read', 'QuickExchange', abilities)) {
+        if (abilitiesSuccess && !CanCan.checkAbilityByAction('read', 'QuickExchange', abilities)) {
```

Comment:
add please abilitiesSuccess to dependency array at line https://github.com/openware/baseapp/blob/5aafee6ad463bde57e5b27839783681343933d8d/web/src/containers/QuickExchange/index.tsx#L97